activemodel/lib/active_model/errors.rb

```diff
@@ -62,6 +62,11 @@ class Errors
     CALLBACKS_OPTIONS = [:if, :unless, :on, :allow_nil, :allow_blank, :strict]
     MESSAGE_OPTIONS = [:message]
 
+    class << self
+      attr_accessor :i18n_full_message # :nodoc:
+    end
+    self.i18n_full_message = false
```

Comment:
Is it added intentionally?

It seems extra since we set default value of `i18n_full_message` in `activemodel/lib/active_model/railtie.rb`.

```diff
diff --git a/activemodel/lib/active_model/errors.rb b/activemodel/lib/active_model/errors.rb
index 56404a036c..7400bb5a6c 100644
--- a/activemodel/lib/active_model/errors.rb
+++ b/activemodel/lib/active_model/errors.rb
@@ -65,7 +65,6 @@ class Errors
     class << self
       attr_accessor :i18n_full_message # :nodoc:
     end
-    self.i18n_full_message = false

     attr_reader :messages, :details

diff --git a/activemodel/lib/active_model/railtie.rb b/activemodel/lib/active_model/railtie.rb
index 0ed70bd473..ed70d65c8e 100644
--- a/activemodel/lib/active_model/railtie.rb
+++ b/activemodel/lib/active_model/railtie.rb
@@ -8,13 +8,14 @@ class Railtie < Rails::Railtie # :nodoc:
     config.eager_load_namespaces << ActiveModel

     config.active_model = ActiveSupport::OrderedOptions.new
+    config.active_model.i18n_full_message = false

     initializer "active_model.secure_password" do
       ActiveModel::SecurePassword.min_cost = Rails.env.test?
     end

     initializer "active_model.i18n_full_message" do
-      ActiveModel::Errors.i18n_full_message = config.active_model.delete(:i18n_full_message) || false
+      ActiveModel::Errors.i18n_full_message = config.active_model.delete(:i18n_full_message)
     end
   end
 end
```

Reply:
Yes, it is intentional. We want a default value if the railtie is not loaded, for example if you use Active Model without Rails.

Reply:
It makes sense then. Thanks for explanation.
What do you think about making `ActiveModel::Errors.i18n_full_message` as   public API in order to allow users use this new feature if use Active Model without Rails?